result/isisiihs.yaml

```diff
@@ -0,0 +1,378 @@
+        - unavailable: The last Graceful Restart status is not initiated.
+      type: object
+      properties:
+        choice:
+          type: string
+          x-field-uid: 1
+          x-enum:
+            succeeded:
+              x-field-uid: 1
+            failed:
+              x-field-uid: 2
+            inprogress:
+              x-field-uid: 3
+            unavailable:
+              x-field-uid: 4
+        succeeded:
+          $ref: '#/components/schemas/IsisIIH.NeighborGRLastAttemptSucceeded'
+          x-field-uid: 2
+        failed:
+          $ref: '#/components/schemas/IsisIIH.NeighborGRLastAttemptFailed'
+          x-field-uid: 3
+        inprogress:
+          type: boolean
+          x-field-uid: 4
+        unavailable:
+          type: boolean
+          x-field-uid: 5
+
+    IsisIIH.NeighborGRLastAttemptFailed:
+      description: >-
+        This container contains the failure status of the last Graceful Restart initiated by this neighbor.
+      type: object
+      properties:
+        reason:
+          description: |-
+            Failure reason of last Graceful Restart.
+          type: string
+          x-field-uid: 1
+
+    IsisIIH.NeighborGRLastAttemptSucceeded:
+      description: >-
+        This object contains the result of a successful graceful restart status in the last attempted by a Neighbor.
+      type: object
+      properties:
+        adjacency_bring_up_time:
+          description: |-
+            The time (in second) is taken to bring up adjacency by the Restarting Neighbor.
+            The time is counted by a Helper router when the Restart TLV with RR bit set in a IIH PDU is received 
+            and till it receives the Restart TLV with RR bit unset in a IIH PDU from the Neighbor Router.
+          type: integer
+          format: uint32
+          x-field-uid: 1
+
+    IsisIIH.NeighborTlvs:
+      description: >-
+        This contains the list of TLVs present in a IIH PDU received from a neighbor IS-IS router.
+      type: object
+      properties:
+        restart_tlv:
+          description: >-
+            Restart Tlv.
+          $ref: '#/components/schemas/IsisIIH.RestartTlv'
+          x-field-uid: 1
+        
+    IsisIIH.RestartTlv:
+      description: >-
+        Container of Restart TLV in IIH PDU.
+        Reference: https://datatracker.ietf.org/doc/html/rfc8706#name-restart-tlv
+      type: object
+      properties:
+        flags:
+          description: >-
+            One octet Restart flags in the Restart TLV. 
+          $ref: '#/components/schemas/IsisIIHRestart.Flags'
+          x-field-uid: 1
+        remaining_time:
+          description: >-
+            Remaining Holding Time (in seconds).
+          type: integer
+          format: uint32
+          x-field-uid: 2
+        restarting_neighbor_id:
+          description: >-
+           Restarting Neighbor's System ID in the format, e.g. '640000000001' in hex format without "0x" at the begining.
```

Comment:
Restarting Neighbor's System ID in hex format without "0x" at the beginning. e.g. '640000000001'